README.md

```diff
@@ -7,6 +7,12 @@ The action runs [golangci-lint](https://github.com/golangci/golangci-lint) and r
 
 ![GitHub Annotations](./static/annotations.png)
 
+## Compatibility
+
+* `v1.2.3` works with `golangci-lint` version >= `v1.28.3`
```

Comment:
not sure if this is still considered as semantic versioning.

The change we made in 1.2.2 is considered as breaking, so having restriction on patch version (e.g. v1.2.3) is not considered in my opinion as backward compatible.

Reply:
Is 1.3.0 good option?
Or we should go with 2.0.0?

Reply:
I personally think 2.0.0 beter. Not sure what others think.